src/memory/objects/character.rs

```diff
@@ -0,0 +1,43 @@
+#[derive(Debug, Default, Clone)]
+pub enum PlayerPartyCharacter {
+    #[default]
+    None,
+    Zale,
+    Valere,
+    Garl,
+    Serai,
+    Reshan,
+    Bst,
+    Moraine,
+    Unknown,
+}
+
+impl PlayerPartyCharacter {
+    // pub fn as_str(&self) -> &'static str {
+    //     match self {
+    //         Self::None => "None",
+    //         Self::Zale => "Zale",
+    //         Self::Valere => "Valere",
+    //         Self::Garl => "Garl",
+    //         Self::Serai => "Seraï",
+    //         Self::Reshan => "Resh'an",
+    //         Self::Bst => "B'st",
+    //         Self::Moraine => "Moraine",
+    //         Self::Unknown => "Unknown",
+    //     }
+    // }
+
+    pub fn parse(value: &str) -> Self {
```

Comment:
iirc this is the strings used internally in memory to identify characters in the party? Maybe add a comment about this.